src/components/editBook.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/style-prop-object */
```

Comment:
- [ ] Kindly ensure that you don't overuse the linter disable. Linters are meant to enforce code quality and too much linter disable means that there is something wrong.